src/cameraGeom/TransformMap.cc

```diff
@@ -259,10 +414,20 @@ std::shared_ptr<TransformMap const> TransformMap::Builder::build() const {
 
     // Return the new TransformMap.
     // We can't use make_shared because TransformMap ctor is private.
-    return std::shared_ptr<TransformMap>(new TransformMap(std::move(frameSet), std::move(frameIds)));
+    return std::shared_ptr<TransformMap>(new TransformMap(std::move(frameSet),
+                                                          std::move(frameIds),
+                                                          std::move(canonicalConnections)));
 }
 
-
 }  // namespace cameraGeom
+
+namespace table {
+namespace io {
+
+template class PersistableFacade<cameraGeom::TransformMap>;
+
+} // namespace io
```

Comment:
Can't find the rule, but usee everywhere else in afw uses 2 spaces before `// namespace`